custom/live-transcription/contexts/TranscriptionProvider.js

```diff
@@ -23,12 +23,12 @@ export const TranscriptionProvider = ({ children }) => {
       // Collect only transcription messages, and gather enough
       // words to be able to post messages at sentence intervals
       if (e.fromId === 'transcription' && e.data?.is_final) {
+        // Get the sender's current display name or the local name
+        const sender =
+          e.data?.session_id !== participants.local.session_id
+            ? participants[e.data.session_id].user_name
```

Comment:
Is there any chance of `e.data` being undefined or null when we get here (since we have optional chaining for it right above) and running into a `TypeError` while trying to access it on this line?